### ISSUE TITLE
关于modular_loss的疑惑

### ISSUE THREAD

**Issue by Ruriiiii01** [justwqq/BMR#1]
您好，最近结合了您的代码拜读了论文Leveraging Brain Modularity Prior for Interpretable Representation Learning of fMRI。有一点问题想要请教一下：modular_loss.py这个文件里面，calculateloss函数返回的是一个np.float类型的值。所以main.py中训练计算batch_loss = criterion(logit, label.to(device)) + argv.lambda1 * reconstruct_loss + argv.lambda2 * modularityloss中，modularityloss也是np.float类型，那么Lm有参与梯度反向传播吗？
